source/Enemy.hx

```diff
@@ -39,8 +39,8 @@ class Enemy extends FlxSprite
 	public override function update(elapsed:Float):Void
 	{		
 		super.update(elapsed);	
-		if (x > 670)
-		x = -30;
+		//if (x > 670)
```

Comment:
You should probably remove this code rather than just commenting it out if this is the final submission